README.md

```diff
@@ -102,13 +102,58 @@ A preview of the VDK CLI commands:
 
 
 # Getting Started
-Installing VDK is a simple pip command. See the [Getting Started](https://github.com/vmware/versatile-data-kit/wiki/getting-started) guide to install VDK and create a data job.
+
+## Create and run data jobs locally
+
+```
+pip install quickstart-vdk
+```
+
+This installs the core vdk packages and the vdk command line interface. You can
+use them to run jobs in your local shell environment.
+
+See also the [Getting
+Started](https://github.com/vmware/versatile-data-kit/wiki/getting-started)
+section of the wiki
+
+## Deploy and manage jobs in Kubernetes cloud
```

Comment:
```suggestion
## Deploy the Control Service locally
```
The format IMO:

> Answer why we need it?
> Prerequisites....
> `vdk server --install`
> 

What the server includes under the hood, doesn't matter - since those are implementation details. 

(1) Based on the prerequisites, Kind is what is needed, so as a proficient user who knows how to work with k8s I'll simply check what namespace gets created iif I want to. It is then obvious we have the CS and Frontend services installed.

(2) Based on the prerequisites, Docker is needed, so in case I want to know I open Docker and see what services are installed. It is quite clear, those are named git server, etc.